packages/client/src/components/Cards/Deck.tsx

```diff
@@ -5,7 +5,18 @@ import { baseCard, CARD_HEIGHT_DESKTOP, CARD_HEIGHT_MOBILE } from './style';
 import { Card as CardType } from 'shared';
 import { Card } from './Card';
 
-export const Deck = ({ cardNumber, scopa, id, ...rest }: { cardNumber: number; scopa?: CardType[] } & BoxProps) => {
+type Props = {
+  cardNumber: number;
+  scopa?: CardType[];
```

Comment:
you need to keep this, even if you have defaults ts will complain because some components aren't passing them

https://www.typescriptlang.org/play?#code/MYewdgzgLgBAhjAvDAFAIwFw2gJwJZgDmSMA5KQJRIB8MA3jKJCADYCmAdCyIelQL4AoJtHgJkKBmhLkY-LFKy4CxflUS0GI1p2680AwYLgoqAejMwQAa2Mm6amBZhscOEDiA